### ISSUE TITLE
mark flag to recognize done or not

### ISSUE THREAD

**Issue by sideroad** [koikijs/kyoden#14]

**Comment by koji-inoue** [comment]
Also it would be cool if we would know whether the pay had been done or not for each transaction.

**Comment by sideroad** [comment]
@koji-inoue yes. We are plan to have flag and it will be separated in each transaction.
